src/index.js

```diff
@@ -0,0 +1,61 @@
+import './style.css';
+import ToDOList from './modules/layout.js';
+import CreateList from './modules/arrlists.js';
+import Storage from './modules/localStorage.js';
+
+document.addEventListener('DOMContentLoaded', ToDOList.displayToDo);
+
+document.getElementById('cleartask').addEventListener('click', () => {
+  Storage.removeCompleted();
+});
+
+document.querySelector('form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const todoL = Storage.getLists();
+  const toDoInput = document.getElementById('todo-input').value;
+  const id = todoL.length + 1;
+  const completed = false;
+
+  const todo = new CreateList(toDoInput, id, completed);
+
+  ToDOList.addLList(todo);
+
+  Storage.addTodo(todo);
+
+  ToDOList.clearField();
+});
+
+// document.getElementById('cleartask').addEventListener('click', () => { });

```

Comment:
Commented-out blocks of code are considered bad practice, so it would be best if you removed them. You don't have to fear losing these pieces of code since they will always be available in your git history.